stdlib/public/core/ImplicitlyUnwrappedOptional.swift

```diff
@@ -49,3 +49,73 @@ public enum ImplicitlyUnwrappedOptional<Wrapped> : ExpressibleByNilLiteral {
     self = .none
   }
 }
+
+extension ImplicitlyUnwrappedOptional : CustomStringConvertible {
```

Comment:
Should we leave this one "removed"? It's not needed to fix the regression at hand.